addons/web_editor/static/tests/html_field_tests.js

```diff
@@ -72,55 +78,55 @@ QUnit.module("WebEditor.HtmlField", ({ beforeEach }) => {
             id: 1,
             txt: htmlDocumentTextTemplate('Hello', 'red'),
         }];
-        const writePromise = makeDeferred();
         await makeView({
             type: "form",
             resId: 1,
             resModel: "partner",
             serverData,
             arch: `
-                <form string="Partner">
+                <form>
                     <sheet>
                         <notebook>
                                 <page string="Body" name="body">
-                                    <field name="txt" widget="html"/>
+                                    <field name="txt" widget="html" options="{'safeHtmlEdit': true}"/>
```

Comment:
I'm confused again. The test now uses the new option, but then we don't test the html field _without_ this option anymore ?